### ISSUE TITLE
Wording of message (minor)

### ISSUE THREAD

**Issue by geocachermgo** [cgeo/cgeo#15312]
### Describe your problem!

If I long-click on the compass symbol of a waypoint without cooordinates I get the messagge "c:geo kennt die Position des Caches nicht".  Correct would be "c:geo kennt die Position des Wegpunktes nicht".
oder "Zu diesem Punkt ist keine Position vorhanden"

### How to reproduce?

long-click on the compass symbol of a waypoint without cooordinates

### Actual result after these steps?

messagge "c:geo kennt die Position des Caches nicht". 

### Expected result after these steps?

Message "Zu diesem Punkt ist keine Position vorhanden"
English "c:geo doesnt know location of point"

### Reproducible

Yes

### c:geo Version

2024.02.11-NB-33a1cac

**Comment by MagpieFourtyTwo** [comment]
What about just changing the translation string to a more general version
like "c:geo does not know the coordinates of this entry" or "current item has no coordinates"?
This way we wouldn't need just another specialized translation plus the source code must not be changed.  ;)

**Comment by eddiemuc** [comment]
What about not showing the compass symbol in such cases?

**Comment by MagpieFourtyTwo** [comment]
Although pretty logical. I would say, the bare hiding of the compass symbol may not be very intuitive.
Cause a user, who is not aware of the fact that his current point has no coordinates, will not gain insight from a missing symbol. He will most probably just assume a bug. ;)  Maybe better to cross the symbol out ...? Or grey it out ...?  Apart from being way more intuitive (IMHO), this would add. give the opportunity to display a tool tip text like "No coordinates available" ...?

**Comment by Lineflyer** [comment]
> What about not showing the compass symbol in such cases?

I learned some time ago, that in most cases it is better UI style to always show all menus/icons in such cases and rather disable them and/or provide user feedback why they are currently not usable.
The benefit is:
- In menu lists users will find the functions always at the same position
- The feedback why its not usable makes users learn. If items are just missing those users might end up on our support.

Same BTW for the personal note upload icon which is hidden for BMs:
It would be better to always show it and provide a feedback like " Uploading your note is only available for PM. Check your membership state or re-login".
Cause quite a few users contact support about this in cases where they are not logged in for some reason....but thats another issue.

**Comment by eddiemuc** [comment]
> I learned some time ago, that in most cases it is better UI style to always show all menus/icons in such cases and rather disable them and/or provide user feedback why they are currently not usable.

Fine for me, but we hide stuff in the GUI all the time in other places. For example, should we then really merge https://github.com/cgeo/cgeo/pull/15442? 

Another example are various options available when tapping on the map. E.g. append/prepend to route or show lines to waypoints or map elevation are only shown under special conditions.

Another example coming to my mind is the possibility to give ratings or favorite points in a log entry - this is also just shown when it is possible to use it.

There is probably much more.

**Comment by MagpieFourtyTwo** [comment]
I agree with @Lineflyer WRT GUI rules (seems as if we learned the same stuff ;) ).

Moreover disabled options can even be a good way to present c:geo's potential to the user. What he can get if ... e. g. he only would be a PM.  Or activated internal routing resp. BRouter.  Or whatever.
IMO c:geo should advertise its possibilities a little more anyway ... there are far too many users out there who don't even know all the basic functions, let alone have any idea what else c:geo is capable to do.

Additionally a button, which is just disabled/greyed out (instead of being completely invisible) because of internal problems can still provide a tool tip or even a popup with the underlying reason for its deactivation. Writing this, the sometimes suddenly vanishing ``Upload personal note`` button comes to my mind.  Would be way better to tell the user that its currently not available because of e. g. connection or authentication problems or because he's no PM than to just hide it, which is often seen as a SW bug.

Mnemonic:  A hidden button can't tell no story.  ;)

**Comment by Lineflyer** [comment]
I have taken out the note upload button UI into a dedicated issue:
https://github.com/cgeo/cgeo/issues/15482
